core/clients/Citizen/src/js/requestData.js

```diff
@@ -0,0 +1,22 @@
+"use strict;"
+
+var $ = require('jquery');
```

Comment:
ok for merge, but let's get rid of this to favor the "serverAPI" approach. See https://github.com/anthill/6element/blob/master/core/clients/Dashboard/src/serverAPI.js